e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_spec.ts

```diff
@@ -30,7 +30,7 @@ describe('AD / LDAP', () => {
         cy.apiRequireLicenseForFeature('LDAP', 'SAML');
 
         // # Create new LDAP user
-        cy.createLDAPUser().then((user) => {
+        (cy as any).createLDAPUser().then((user) => {
```

Comment:
this is not an issue once you update to latest master

Reply:
updated